datamade/la-metro-councilmatic
# Add Tim Sandoval headshot and bio, documentation for adding new board members

## Overview

See title.

### Checklist


## Testing Instructions

The headshot and bio are keyed off the production person slug, so they won't be visible in local development or on staging.

 * View the revised README and confirm the Markdown looks good.
 * Review the instructions to add a new member, and confirm they make sense.
 * Double check that the bio and headshot have been added, as documented in the README.

Handles #693
